### PR TITLE
Finish documenting outer classes

### PR DESCRIPTION
**Pull request by msmow**
Document all the various handlers that are used in main, including utils.
Add usage of the help function when launching each file, showcasing all methods and attributes of every class.

### CODE REVIEW

File: realsense.py [hunk 1]
```diff
@@ -49,22 +78,27 @@ def frame(self):
...
         return 424, 240
 
     def set_exposure(self, exposure: int):
         """
-        Set the exposure to a desired value.
-        :param exposure:
-
+        Set the exposure to a desired value. May not set the camera to the exact value, so the actual exposure of the
+        camera is logged.
+        :param exposure: Exposure to se the camera to.
```
Comment (pklito) [inline realsense.py]:
```suggestion
        :param exposure: Exposure to set the camera to.
```

**Review by pklito — CHANGES_REQUESTED** [review]

File: main.py [hunk 2]
```diff
@@ -67,9 +70,14 @@ def get_args():
     parser.add_argument('-no-web', action='store_false', default=True,
                         dest='web',
                         help='Disable web server UI')
+    # Add networktables argument
     parser.add_argument('-networktables', '-nt', action='store_true', default=False,
                         dest='networktables',
                         help='Initiate network tables')
+    # Add help in main argument
```
Comment (orel-david) [inline main.py:77]: What does help mean ?

File: main.py [hunk 3]
```diff
@@ -255,5 +263,7 @@ def loop(self):
 
 
 if __name__ == '__main__':
-    help(Main)
-    Main().loop()
+    main = Main()
+    if main.results.help_main:
+        help(main)
+    main.loop()
```
Comment (orel-david) [inline main.py:269]: I think this if statement needs some documentation

**Review by orel-david — COMMENTED** [review]
The documentation is clear and except this one if statement everything is pretty clear

**Pull request merged**